### ISSUE TITLE
[Import from macOS repo] Foreign keys dropdown

### ISSUE THREAD

**Issue by huyphams** [TablePlus/TablePlus-Windows#220]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): PostgreSQL?

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 122

3. The steps to reproduce this issue:

1. Open a table.
2. Set value to the foreign key with dropdown???

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by ujaeggi** [comment]
Hoi Huy

I use TablePlus under macOS. The intention was a request for a new feature. I added screenshots from HeidiSQL as inspiration how it is solved there. (HeidiSQL is only available for Windows)

Or does that already work with TablePlus?

**Comment by huyphams** [comment]
Thanks for confirming. You removed all the issue template questions without answering it so I thought you're using Windows (because you've mentioned HeidiSQL). Sorry about that.

**Comment by aminoxsolami** [comment]
Hi HiyPhams, I'm just wondering if you are the publisher of video saver app on sellmyapp, I want to add some features and I need you contact.

Thank you

**Comment by ujaeggi** [comment]
> Thanks for confirming. You removed all the issue template questions without answering it so I thought you're using Windows (because you've mentioned HeidiSQL). Sorry about that.

True. I guessed it‘s not needed because it‘s not an issue or related to any version. Sorry
